### ISSUE TITLE
After an /ircreload, messages are doubled in the Staff channel

### ISSUE THREAD

**Issue by WizardCM** [WMCAlliance/BukkitIRCd#39]
Quite self explanatory.

<Minecraft> Survival Owner Wizard is no longer away.
<Minecraft> Survival Owner Wizard is no longer away.
<Minecraft> Survival Owner Wizard is now away.
<Minecraft> Survival Owner Wizard is now away.

**Comment by chyyran** [comment]
This happens to me too, might have something to do with 2 IRCd instances being alive after a reload

**Comment by WizardCM** [comment]
Something I've noticed is that the first/previous instance of IRCd dies within a minute or two, because this issue cleans itself up. Maybe we should just add a warning to the /ircreload message?

**Comment by chyyran** [comment]
Yeah, maybe that just be fine. A part of me does want to get this fixed though, but I'm busy working on another project atm.
